glossary.md

```diff
@@ -13,52 +13,73 @@ gh: specifically for github
+| git pull | This basically does the work of git fetch and git merge in one command. It's when you know fosho that your friends code is good and you want to add it to your repository. | 
+|git add [filename] | This command adds the specific changed file you're referencing to in [filename] to the staging area. You're adding your new notes/changes into the tempoary section of the binder. |
+|git add . | This commands all the files that you changed to the staging area. The '.' is what makes it ALL the files you've changed | 
+| git push [alias][branch] | Now you've coded a function and you want to add that to the online repo. You use git push to add your code. Sometimes, you have to make a PR (pull request).  | 
+| git commit -m "[message]" | commits your current content as a new commit with a message to add context. Whenever you commmit, it's a good idea to add a message for context to explain what you're commiting |
+|git head| Similar to a bookmark telling you which note you're on, git head tells you which commit you're working on. |
+
+The head is referred to as the current commit you are working on. 
+
 
 
-To simplify it, git pull is a combo of fetch+merge.
+**PULL REQUEST (PR)**: Based on the permissions of your repository, you can't always just immediately add your code. A pull request is when you push your changes to a different branch and you open a 'pull request' on the github website so someone can review your code before it's added into the repository. It's a good way of checking each other and not just messing up the code.
 
-### Branches
+### History & Branches
 
-Github uses *branches* to create seperate lines of development in a repository. It keeps it clean, organized, and allows for fixing bugs or adding features. Multiple developers can work on 1 project simultaneously without messing with each other's work. 
+Github uses *branches* to create seperate lines of development in a repository. Let's say we want each team to work on seperate parts of the project without interfering with each other's work. We create separate sections for each member's contributions. It keeps it clean, organized, and allows for fixing bugs or adding features.
+	The *Main* branch is the branch with all the finalized code and details
 
 | command    | use |
 | -------- | ------- |
-| git branch -a | see all the branches (local & remote). The branch you are on is marked with a *    |
-|git branch [branch-name] | create a new branch at the current commit |
-|git checkout [branch-name] | switch to another branch |
-|git merge [branch] | merge the branch into your current one | 
-| git log | show all commits in your current branch's history |
+| git branch -a | see all the branches (local & remote). The branch you are currently working is marked with a *    |
+|git branch [branch-name] | create a new branch at the current commit. Basicaly, it's like adding a new section at the same place you're working on. It does not change the current section |
```

Comment:
Nice inclusion, might be good to explain the difference between branch, switch, and checkout
https://stackoverflow.com/questions/57265785/whats-the-difference-between-git-switch-and-git-checkout-branch
https://stackoverflow.com/questions/7987687/what-is-the-difference-between-git-branch-and-git-checkout-b

Reply:
also spelling of "basically"